homeassistant/components/opower/coordinator.py

```diff
@@ -93,7 +93,7 @@ async def _insert_statistics(self) -> None:
                 (
                     self.api.utility.subdomain(),
                     account.meter_type.name.lower(),
-                    account.utility_account_id,
+                    account.utility_account_id.replace("-", "_"),
```

Comment:
Would be good to add a comment about why we are mutating this value.

Since it didn't work before I don't think its a breaking change

Reply:
Done. Correct this isn't a breaking change. This is only needed for the new AEP utilities that fail with HomeAssistantError("Invalid statistic_id") because they have dashes in their account ids. See valid_entity_id function in core.py.